src/webextension/manage/actions.js

```diff
@@ -133,6 +137,7 @@ export function removeItem(id) {
       id,
     });
     dispatch(removeItemCompleted(actionId, id));
+    telemetry.recordEvent("lockbox", "itemDeleting", "updatingItemForm");
```

Comment:
Unlike the other "-ing" events, this is getting fired *after* deletion completes, instead of when it's requested. For consistency, I'd move this up to just below `dispatch(removeItemStarting(actionId, id));`